Improve download section

burodepeper opened the pull request.

chrissimpkins commented:
Thanks David.  Title says in progress.  Let me know when you are done with work and I will get this merged in.

burodepeper commented:
The wording could probably be improved upon, but I think this makes the download process much clearer to an average user. I've decided not to implement any active OS detection. I think the short intro text and the two buttons make things very clear, and this method still leaves the choice for Windows users to opt for the ttf files. I've also updated the bits about the playground and using the web font cdn. 

Probably least conspicuous, but I've removed the "try it" option from the main menu up top. I find it confusing next to "download". The links to "specimen" and "source" could benefit from an added icon that suggests that you will be leaving the current page. A navigation bar usually suggests that you are navigating within the current website, but we can save the for later.

chrissimpkins commented:
Looks great David.  I added the Windows installer links to the footer link tables on both the index.html and playground pages.  Thanks much!

cc: @texhex
